papergres.go

```diff
@@ -436,89 +447,90 @@ func (s *Schema) InsertAll(objs interface{}) ([]*Result, error) {
 		return nil, err
 	}
 	if len(slice) == 0 {
-		return nil, errors.New("Empty slice")
+		return nil, errors.New("empty slice")
```

Comment:
Why lowercase?

Reply:
It's a standard in Go to always have error strings not capitalized.
For reference, read here: https://github.com/golang/go/wiki/CodeReviewComments#error-strings